cmd/agent/main.go

```diff
@@ -29,6 +30,12 @@ import (
 
 func main() {
 	ctx := controllerruntime.SetupSignalHandler()
+	// Starting from version 0.15.0, controller-runtime expects its consumers to set a logger through log.SetLogger.
+	// If SetLogger is not called within the first 30 seconds of a binaries lifetime, it will get
+	// set to a NullLogSink and report an error. Here's to silence the "log.SetLogger(...) was never called; logs will not be displayed" error
+	// by setting a logger through log.SetLogger.
+	// More info refer to: https://github.com/karmada-io/karmada/pull/4885.
```

Comment:
Just a question, whether the url needs to be updated.
others LGTM

Reply:
update to what?

Reply:
https://github.com/karmada-io/karmada/pull/5100

Reply:
I don't think so, current PR don't have more info to refer.